auth0/src/test/java/com/auth0/android/result/UserProfileTest.java

```diff
@@ -41,7 +40,7 @@ public void getId() {
     @SuppressWarnings("unchecked")
     @Test
     public void shouldReturnSubIfMissingId() {
-        Map<String, Object> extraInfo = Collections.singletonMap("sub", "fromSub");
+        Map<String, Object> extraInfo = Collections.singletonMap("sub", (Object) "fromSub");
```

Comment:
This cast is redundant, why is it added?

Reply:
Looks redundant but it's actually required for the file to compile. This was needed after I fixed the import above.

Reply:
Updated to not cast.